dto/decision_dto.go

```diff
@@ -62,6 +63,7 @@ type APIDecision struct {
 	Rules                []APIDecisionRule   `json:"rules"`
 	Score                int                 `json:"score"`
 	ScheduledExecutionId *string             `json:"scheduled_execution_id,omitempty"`
+	ScenarioIterationId  string              `json:"scenario_iteration_id"`
```

Comment:
I would rather do this until we have backfilled existing values
```suggestion
	ScenarioIterationId  string              `json:"scenario_iteration_id,omitempty"`
```

Reply:
ok, can do 👍